docs/admin/install/troubleshoot.rst

```diff
@@ -31,7 +31,7 @@ On some hardware, network devices (Ethernet and Wi-Fi) will not immediately work
 
 |screenshot_sys_net_pci_reset|
 
-Open a ``dom0`` terminal via **Q > Settings Gear > Other Tools > Xfce Terminal**, and run the following command to list the devices connected to the ``sys-net`` VM.
+Open a ``dom0`` terminal via **Q > Gear Icon (left-hand side) > Other Tools > Xfce Terminal**, and run the following command to list the devices connected to the ``sys-net`` VM.
```

Comment:
As was discussed in #228, unfortunately there are two settings gears visible at the same time in the menu, so we have to disambiguate.